README.md

```diff
@@ -49,7 +49,46 @@ The project follows BEM methodology with the following structure:
 - Add photo upload capability
 - Create user authentication
 
```

Comment:
* Please, make the modals match the design https://snipboard.io/HIkaDb.jpg 
* The `confirmation` modal doesn't close when I click on the `overlay`
* The `confirmation` modal should match the design
* Please, add some `margins` between the `avatar` and the profile block https://snipboard.io/lSXWZD.jpg